example_config_class.rb

```diff
@@ -13,8 +13,6 @@ def self.process(message)
   config.logger = Logger.new
   config.preprocessor = ->(_) { FastlyNsq.logger.info 'PREPROCESSESES' }
 
-  config.listener_config do |lc|
-    lc.add_topic('some_topic', MessageProcessor)
-    lc.add_topic('some_other_topic', MessageProcessor)
-  end
+  lc.listen('some_topic', MessageProcessor)
+  lc.listen('some_other_topic', MessageProcessor, priority: 7)
```

Comment:
It would be nice to suggest the usage of a lambda here now that it is possible to do that. Mostly because it is nice and this is OO :P
```
lc.listen('some_puts_topic', -> (msg) { puts msg }, priority: 7)
```

Reply:
I'll add it to the README